code/modules/cm_aliens/weeds.dm

```diff
@@ -12,6 +12,11 @@
 	unacidable = 1
 	health = 1
 
+/obj/effect/alien/weeds/healthcheck()
+    if(health <= 0)
+        round_statistics.weeds_destroyed++
```

Comment:
Add this to Dispose()/Destroy().
Explosions & etcetera can straight out skip healthcheck() and delete them.

Reply:
Uh, now we need a reminder to find out the source of this statistic problem.

Reply:
it's probably something about "can this weed go into a wall? no? delete the weed." not really an issue cause no one has ever noticed before now.